test/Microsoft.Azure.Devices.Edge.Test.Common/EdgeModule.cs

```diff
@@ -39,57 +39,44 @@ public static Task WaitForStatusAsync(EdgeModule[] modules, EdgeModuleStatus des
+                                {
+                                    var columns = ln.Split(null as char[], StringSplitOptions.RemoveEmptyEntries);
+                                    foreach (var module in modules)
                                     {
-                                        var columns = ln.Split(null as char[], StringSplitOptions.RemoveEmptyEntries);
-                                        foreach (var module in modules)
+                                        // each line is "name status"
+                                        if (columns[0] == module.Id &&
+                                            columns[1].Equals(desired.ToString(), StringComparison.OrdinalIgnoreCase))
                                         {
-                                            // each line is "name status"
-                                            if (columns[0] == module.Id &&
-                                                columns[1].Equals(desired.ToString(), StringComparison.OrdinalIgnoreCase))
-                                            {
-                                                return true;
-                                            }
+                                            return true;
                                         }
+                                    }
 
-                                        return false;
-                                    }).ToArray();
-                        },
-                        a => a.Length == modules.Length,
-                        e =>
-                        {
-                            // Retry if iotedged's management endpoint is still starting up,
-                            // and therefore isn't responding to `iotedge list` yet
-                            bool DaemonNotReady(string details) =>
-                                details.Contains("Could not list modules", StringComparison.OrdinalIgnoreCase) ||
-                                details.Contains("Socket file could not be found", StringComparison.OrdinalIgnoreCase);
+                                    return false;
+                                }).ToArray();
+                    },
+                    a => a.Length == modules.Length,
+                    e =>
+                    {
+                        // Retry if iotedged's management endpoint is still starting up,
+                        // and therefore isn't responding to `iotedge list` yet
+                        bool DaemonNotReady(string details) =>
+                            details.Contains("Could not list modules", StringComparison.OrdinalIgnoreCase) ||
+                            details.Contains("Socket file could not be found", StringComparison.OrdinalIgnoreCase);
 
-                            return DaemonNotReady(e.ToString());
-                        },
-                        TimeSpan.FromSeconds(5),
-                        token);
-                }
-                catch (OperationCanceledException)
```

Comment:
why removing these custom exception message?

Reply:
It was inconsistent...there are **lots** of places when the OperationCanceledException can be thrown, and we were adding a custom message to only a few of them. So I removed the one-offs and added a single custom message in Test.cs (which will probably be refactored into a better location when more tests are added).